client/my-sites/sharing/connections/service-placeholder.jsx

```diff
@@ -3,6 +3,8 @@
  */
 var React = require( 'react' );
 
+var Button = require( 'components/Button' );
```

Comment:
We need to use the lowercase Button here here. `require( 'components/button' );` 
react is throwing me some warnings. 
![screen shot 2016-01-25 at 12 23 59](https://cloud.githubusercontent.com/assets/115071/12563315/8be026f0-c35e-11e5-9560-131880a85c42.png)